roles/openshift_gcp/tasks/configure_master_bootstrap.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Copy unit service
```

Comment:
What does this `configure_master_bootstrap.yml` do?  What are these timers doing?

Reply:
The timers refresh the bootstrap credentials onto the VM every X unit interval from project metadata.